BenchPress/Generators/ResourceTypes/ApiManagementApi.cs

```diff
@@ -0,0 +1,23 @@
+namespace Generators.ResourceTypes;
+
+public class ApiManagementApi : ResourceType
+{
+    public ApiManagementApi() { }
+
+    public override string Id => "Microsoft.ApiManagement/service/apis";
+    public override string FullName => Id;
+    public override string FriendlyName => "API Management Service APIs";
+    public override string Prefix => "amsapi";
+    public override string FunctionPrefix => "ApiManagementApi";
+
+    public override IEnumerable<KeyValuePair<string, object>> GetResourceParameters(TestMetadata m)
+    {
+        return new[]
+        {
+            Param("ResourceType", "ApiManagementApi"),
+            Param("ResourceName", m.ResourceName),
+            Param("ResourceGroupName", m.ExtraProperties["resourceGroup"]),
+            Param("ServiceName", m.ExtraProperties["serviceName"])
```

Comment:
The type of the "parent" resource is Microsoft.ApiManagement/service. So shouldn't the key be `service`?

Reply:
It's `serviceName` in the generated ARM template.

Reply:
All of the key names should be fixed!

Thanks for walking me through that. It took me a while, but I finally got it!